.github/workflows/yamllint.yaml

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
We can't use https://github.com/lsst/rubin_workflows/blob/main/.github/workflows/yamllint.yaml ?  Should be similar to the rebase_checker.